recipes-bsp/formfactor/formfactor/generic-armel-iproc/machconfig

```diff
@@ -0,0 +1,3 @@
+# Assume a USB mouse and keyboard are connected
```

Comment:
Should the comment say 'not connected'?
Or does the HAVE_KEYBOARD refer to a non-USB keyboard?

```suggestion
# Assume a USB mouse and keyboard are not connected
```